src/components/Completed.js

```diff
@@ -1,13 +1,19 @@
 import React from 'react';
+import progress from '../img/progress.png';
+import '../style/completed.css';
 
 const Completed = () => (
-  <div className="completed">
-    <h2>
-      0%
-      <br />
-      <span>Completed</span>
-    </h2>
-  </div>
+  <>
+    <div className="book-completed">
```

Comment:
- [ ] Kindly, use the same height on this section :point_down: 

![bb5](https://user-images.githubusercontent.com/57562869/155612535-705defb7-525a-4533-9337-e465b81474d4.png)

Reply:
Kidly fix linter configuration error